setup.cfg

```
@@ -25,7 +25,7 @@ zip_safe = False
 packages = find:
 install_requires=
     gitpython>3
-    dulwich>=0.20.38
+    dulwich<=0.20.40 # set upper bound for dulwich so that we can check that wheels are available before bumping version
```

```suggestion
    dulwich<=0.20.38
```
dulwich wheels are now available